src/Adapter/Employee/ContextEmployeeProvider.php

```diff
@@ -52,6 +52,10 @@ public function __construct(Employee $contextEmployee)
      */
     public function isSuperAdmin()
     {
+        if (null === $this->contextEmployee) {
+            return false;
```

Comment:
I'd prefer not to return `false` here, this service ideally should only be used at controller layer, if it's being used elsewhere (e.g. command line), then exception should be thrown.